docs/dns/rrsets.rst

```diff
@@ -646,21 +661,29 @@ Record types with priority field
       undefined, per RFC 4592, Sec. 4.2.
 
 ``TXT`` record
-    The contents of the ``TXT`` record must be enclosed in double quotes.
-    Thus, when ``POST``\ ing to the API, make sure to do proper escaping etc.
-    as required by the client you are using.  Here's an example of how to
-    create a ``TXT`` RRset::
-
-        curl -X POST https://desec.io/api/v1/domains/{name}/rrsets/ \
-            --header "Authorization: Token {token}" \
-            --header "Content-Type: application/json" --data @- <<< \
-            '{"type": "TXT", "records": ["\"test value1\"","\"value2\""], "ttl": 3600}'
-
-    Binary record contents are supported, but subject to various escaping
-    rules (both JSON and ``TXT`` record syntax; in addition, certain
-    non-printable characters are not accepted even when unicode-escaped, like
-    ``\u0000``).  Still, you can store any binary data by using DNS-style
-    ``\DDD`` encoding for your binary data (see RFC 1035 Sec. 3.3.14 and 5.1).
-    For example, a carriage return (``\r``) can be stored as ``\013``.  (Note
-    that JSON encoding needs to be applied on top of that, so a valid
-    ``records`` field would be ``["\"\\013\""]``.)
+    - The contents of the ``TXT`` record must be enclosed in double quotes.
+      Thus, when ``POST``\ ing to the API, make sure to do proper escaping
+      etc. as required by the client you are using.  Here's an example of how
+      to create a ``TXT`` RRset::
+
+          curl -X POST https://desec.io/api/v1/domains/{name}/rrsets/ \
+              --header "Authorization: Token {token}" \
+              --header "Content-Type: application/json" --data @- <<< \
+              '{"type": "TXT", "records": ["\"test value1\"","\"value2\""], "ttl": 3600}'
+
+    - Record values consisting of several token strings separated by
+      whitespace are permitted, for example:
+      ``["\"token 1 of record 1\" \"token 2 of record 1\"", "\"record 2\""]``
+
+      Token strings longer than 255 characters are automatically split into
+      several token strings.  (The maximum length is given by the DNS
```

Comment:
clarify: *This* maximum length is given by

Reply:
done